pkg/client.go

```diff
@@ -1212,21 +1212,28 @@ func (o *Client) ValidateAPIKey(apiKeyToken string) (*models.APIKeyValidation, e
 // public methods for misc functionality
 
 // CreateAccessToken creates an access token.
-func (o *Client) CreateAccessToken(userID uuid.UUID, durationInMinutes int) (*models.AccessToken, error) {
+func (o *Client) CreateAccessToken(userID uuid.UUID, durationInMinutes int, createAccessTokenOptions ...models.CreateAccessTokenOptions) (*models.AccessToken, error) {
 	urlPostfix := "access_token"
 
 	// assemble body params
 
 	type CreateAccessToken struct {
-		UserID            uuid.UUID `json:"user_id"`
-		DurationInMinutes int       `json:"duration_in_minutes"`
+		UserID               uuid.UUID  `json:"user_id"`
+		DurationInMinutes    int        `json:"duration_in_minutes"`
+		ActiveOrgId          *uuid.UUID `json:"active_org_id,omitempty"`
+		WithActiveOrgSupport *bool      `json:"with_active_org_support,omitempty"`
 	}
 
 	bodyParams := CreateAccessToken{
 		UserID:            userID,
 		DurationInMinutes: durationInMinutes,
 	}
 
+	if len(createAccessTokenOptions) == 1 {
```

Comment:
Note: The reason we do this is as to not break backwards compatibility -- otherwise, everyone will get an error due to not having parameters for the new parameters. 

Smoke tested it as such:

```
accessToken, _ := client.CreateAccessToken(uuid.MustParse("c1d45787-b8b8-439d-9203-4af9067788bd"), 60*24, models.CreateAccessTokenOptions{
		ActiveOrgId: <value, can leave this whole param out>,
		WithActiveOrgSupport: <value, can leave this whole param out>,
	})
```